File: src/main/java/com/senderman/stickersorterbot/bot/MessageSender.java
package com.senderman.stickersorterbot.bot;

import com.annimon.tgbotsmodule.api.methods.send.SendMessageMethod;
import org.telegram.telegrambots.meta.api.objects.Message;

public interface MessageSender {

    Message sendMessage(long chatId, String text);

    Message sendMessage(SendMessageMethod sm);

}

File: src/main/kotlin/com/senderman/stickersorterbot/bot/MessageSender.kt
package com.senderman.stickersorterbot.bot

import com.annimon.tgbotsmodule.api.methods.send.SendMessageMethod
import org.telegram.telegrambots.meta.api.objects.Message

interface MessageSender {

    fun sendMessage(chatId: Long, text: String, replyToMessageId:Int?=null):Message

    fun sendMessage(sm:SendMessageMethod):Message

}